mllib/src/test/scala/org/apache/spark/ml/image/ImageSchemaSuite.scala

```diff
@@ -17,18 +17,19 @@
 
 package org.apache.spark.ml.image
 
+import java.io.File
 import java.nio.file.Paths
 import java.util.Arrays
 
 import org.apache.spark.SparkFunSuite
 import org.apache.spark.ml.image.ImageSchema._
 import org.apache.spark.mllib.util.MLlibTestSparkContext
 import org.apache.spark.sql.Row
-import org.apache.spark.sql.types._
 
 class ImageSchemaSuite extends SparkFunSuite with MLlibTestSparkContext {
   // Single column of images named "image"
-  private lazy val imagePath = "../data/mllib/images/origin"
+  private lazy val imagePath = new File(this.getClass.getResource("/").getPath
```

Comment:
This doesn't look right; looks like you're trying to fix the test rather than you're env. The tests pass when run in my IDE and pass when run in the build. I'd close this.

Reply:
The ubuntu and Mac  system both have this problem.  Are there any configuration except "git clone spar, mvn clean -DskipTests install, then run the test case"?

Reply:
I use Mac but looks no problem in my IDE. Jenkins looks no problem too. Can you investigate why it is?

Reply:
ok @HyukjinKwon